libs/clients/taktikal/README.md

```diff
@@ -0,0 +1,38 @@
+# Taktikal
+
+## About
+
+The client allows its users to request individuals to sign PDF documents using their mobile electronic ids via Taktikal.
+
+## Usage
```

Comment:
I think it would be good to update the documentation on how to use the `TaktikalService` to in your services to sign documents.